<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportFromSailDialog</class>
 <widget class="QDialog" name="ImportFromSailDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Импорт из ИС &quot;Парус&quot;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="grpImportOrgStructure">
     <property name="title">
      <string>Подразделение</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="margin">
       <number>4</number>
      </property>
      <property name="spacing">
       <number>4</number>
      </property>
      <item row="2" column="0">
       <widget class="QLineEdit" name="edtFileNameOrgStructure"/>
      </item>
      <item row="2" column="1">
       <widget class="QToolButton" name="btnSelectFileOrgStructure">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="btnViewOrgStructure">
        <property name="text">
         <string>Просмотреть</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="chkTypeOrgStructure">
        <property name="text">
         <string>Заполнять тип подразделения</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QCheckBox" name="chkHasHospitalBeds">
        <property name="text">
         <string>Заполнять &quot;Имеет койки&quot;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Интерпретировать код подразделения как</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QComboBox" name="cmbOrgStructureCodeType">
        <item>
         <property name="text">
          <string>Код основной</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Код по бухгалтерии</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="chkUpdateOrgStructure">
        <property name="text">
         <string>Обновлять</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="chkAttachOrgStructure">
        <property name="text">
         <string>Добавлять</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Режим импорта</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" rowspan="2" colspan="3">
    <widget class="QGroupBox" name="grpImportPerson">
     <property name="title">
      <string>Персонал</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="margin">
       <number>4</number>
      </property>
      <property name="spacing">
       <number>4</number>
      </property>
      <item row="4" column="0">
       <widget class="QLineEdit" name="edtFileNamePerson"/>
      </item>
      <item row="4" column="1">
       <widget class="QToolButton" name="btnSelectFilePerson">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="btnViewPerson">
        <property name="text">
         <string>Просмотреть</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QComboBox" name="cmbSpecialityCodeType">
        <item>
         <property name="text">
          <string>Код ОКСО</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Региональный код</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Интерпретировать поле SPECIALITY как</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Интерпретировать поле POST_ID как</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QComboBox" name="cmbPostCodeType">
        <item>
         <property name="text">
          <string>Код ОКДП</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Региональный код</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cmbOrgstructureId">
        <item>
         <property name="text">
          <string>Да</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Нет</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Код подразделения является идентификатором</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QProgressBar" name="prbImport">
     <property name="value">
      <number>24</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="3">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnImport">
       <property name="text">
        <string>начать импортирование</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>закрыть</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QLabel" name="stat">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QTextBrowser" name="log"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
